python/lsst/ts/ledprojector/led_controller.py

```diff
@@ -845,7 +845,13 @@ def _blocking_connect(self) -> None:
         # Read each input channel, to make sure the configuration is valid.
         input_channel_names = set(lbc.channel for lbc in self.channels.values())
         num_frames = len(input_channel_names)
-        values = ljm.eReadNames(self.handle, num_frames, input_channel_names)
+        for _ in range(3):
+            try:
+                values = ljm.eReadNames(self.handle, num_frames, input_channel_names)
```

Comment:
For some instruments, you might need to add some sleeping time before the next attempt of reading data.
And for some instruments, you do not need to do so.
I am not sure this controller would need some sleeping time or not.
Did you experiment this already?

Reply:
Seems like it's okay without sleeping.